design-reviews/trade-studies/readme.md

```diff
@@ -0,0 +1,49 @@
+# Trade Studies
+
+Trade studies are a tool for selecting the best option out of several possible options for a given problem (for example: compute, storage).
+They evaluate potential choices against a set of objective criteria/customer requirements to clearly lay out the benefits and limitations
+of each solution.
+
+[Trade studies](https://en.wikipedia.org/wiki/Trade_study) are a concept from systems engineering that we adapted for CSE projects. Trade
+studies have proved to be a critical tool to drive alignment with the customers, earn credibility while doing so and ensure our decisions
+were backed by data and not bias.  
+
+## When to use the tool
+
+Trade studies go hand in hand with high level architecture design. This usually occurs as project requirements are solidifying, before
+coding begins. Trade studies continue to be useful throughout the project at any time where there are multiple options that need
+to be selected from. New decision point could occur from changing requirements, getting results of a research spike, or identifying
+challenges that were not originally seen.
+
+Trade studies should be avoided if there is a clear solution choice. Because they require each solution to be fully thought out, they
+have the potential to take a lot of time to complete. When there is a clear design, the trade study should be omitted and an entry
+should be made in the [Decision Log](../decision-log/readme.md) documenting the decision.
+
+## Why Trade Studies
+
+The biggest advantage of using trade studies is formalizing the decision-making process in a clear and objective manner. This gives a clear
+flow for presenting the information to decision makers. We recommend the following approach for sharing information:
+
+|Sharing Approach|Customer Cloud Maturity|Content Presentations|Tools Used fo Writing/Sharing|
```

Comment:
I am not sure what this is trying to suggest. What exactly does `Lightweight` vs `Full Experience` mean? Is this explained somewhere else?

Reply:
I rewrote this section to be a bit clearer and focus on the benefits that trade studies give. I think the new version is clearer